docs/modules/operation/pages/flows/sentinel/message-broker/kafka.adoc

```diff
@@ -29,9 +29,9 @@ config:property-set http-url http://core-instance-ip:8980/opennms<3>
 config:update
 ----
 
-<1> A location string is required and is used only for tracing
-<2> Unique identifier is used as service name only for tracing
-<3> Base URL for the web UI which provides the REST endpoints
+<1> A location string is required and used only for tracing.
+<2> Unique identifier used as service name only for tracing.
```

Comment:
Do we want to note that this can be a hostname label instead of a UUID string? (Same comment applies to AMQ page.

Reply:
Yes that makes sense.

Reply:
Have updated. Let me know if that's what you had in mind.